Tests/scripts/check_base_branch_for_external_pr.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import sys
+from Tests.test_utils import print_error, print_warning, print_color, LOG_COLORS
+
+
+def get_base_branch(pr_num):
```

Comment:
this function seems long, and it has code duplication, you can add two more functions to make it a lot more clear and shorter code.